web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx

```diff
@@ -80,13 +80,17 @@ const AnalysisRunnerSummary: React.FunctionComponent = () => {
         ])
     }
 
+    // console.table(data?.project.analyses)
+
     const flatData = data?.project.analyses.map(({ author, id, output, meta }, i) => ({
         email: author,
         id,
         output,
         ...meta,
         position: i,
-        'Hail Batch': meta?.batch_url.replace('https://batch.hail.populationgenomics.org.au/', ''),
+        'Hail Batch': meta?.batch_url
+            ? meta?.batch_url.replace('https://batch.hail.populationgenomics.org.au/', '')
```

Comment:
Do we have a constants file, if so can we put this link in there?

Reply:
Not yet, but will keep this in mind if we end up with a pile of constants in the future